go.mod

```diff
@@ -26,14 +25,17 @@ require (
 	golang.org/x/text v0.3.2 // indirect
 	gopkg.in/check.v1 v1.0.0-20190902080502-41f04d3bba15 // indirect
 	gopkg.in/yaml.v2 v2.2.4 // indirect
-	k8s.io/api v0.0.0-20190409021203-6e4e0e4f393b
-	k8s.io/apimachinery v0.0.0-20190404173353-6a84e37a896d
+	k8s.io/api v0.0.0-20190918195907-bd6ac527cfd2
+	k8s.io/apimachinery v0.0.0-20190817020851-f2f3a405f61d
 	k8s.io/client-go v11.0.1-0.20190409021438-1a26190bd76a+incompatible
 	k8s.io/cluster-bootstrap v0.0.0-20190516232516-d7d78ab2cfe7
 	k8s.io/klog v0.4.0
 	k8s.io/kube-openapi v0.0.0-20190816220812-743ec37842bf // indirect
-	sigs.k8s.io/cluster-api v0.2.3
-	sigs.k8s.io/controller-runtime v0.2.2
+	sigs.k8s.io/cluster-api v0.2.5
+	sigs.k8s.io/controller-runtime v0.3.0
 )
 
-replace sigs.k8s.io/cluster-api => sigs.k8s.io/cluster-api v0.2.3
+replace (
+	k8s.io/client-go => k8s.io/client-go v0.0.0-20190918200256-06eb1244587a
```

Comment:
If we update the client-go version above, do we still need a replace?

Reply:
Everytime I've tried to remove this previously it reverts back to a different version and introduces build failures.

Reply:
K. If the version ends up not being `v0.0.0-20190918200256-06eb1244587a`, definitely keep it. But we should update the version up above too, to match, right?

Reply:
Actually, it looks like it's working now, removed both.